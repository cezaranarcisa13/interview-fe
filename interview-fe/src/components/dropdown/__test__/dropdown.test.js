import React from "react";
import ReactDOM from "react-dom";
import Dropdown from "../dropdown";
import renderer from "react-test-renderer";

it("renders witout crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Dropdown />, div);
})

it("match snapshot", () => {
    const tree = renderer.create(<Dropdown error={"error"} handleChange={()=>{}} user={"11"} values={[{label:"11", value:"11"}]}/>).toJSON();
    expect(tree).toMatchSnapshot();
})