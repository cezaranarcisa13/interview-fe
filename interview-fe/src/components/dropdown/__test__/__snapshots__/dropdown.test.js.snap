// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
Array [
  <label
    className="MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-root MuiFormLabel-colorPrimary css-9npbnl-MuiFormLabel-root-MuiInputLabel-root"
    id="demo-multiple-name-label"
  >
    User
  </label>,
  <div
    className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
    onClick={[Function]}
  >
    <div
      aria-expanded="false"
      aria-haspopup="listbox"
      aria-labelledby="demo-multiple-name-label demo-multiple-name"
      className="MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input css-11u53oe-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
      id="demo-multiple-name"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      role="button"
      tabIndex={0}
    >
      11
    </div>
    <input
      aria-hidden={true}
      className="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
      onAnimationStart={[Function]}
      onChange={[Function]}
      tabIndex={-1}
      value="11"
    />
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
      data-testid="ArrowDropDownIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 10l5 5 5-5z"
      />
    </svg>
    <fieldset
      aria-hidden={true}
      className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
    >
      <legend
        className="css-173wfxe"
      >
        <span>
          User
        </span>
      </legend>
    </fieldset>
  </div>,
  <p
    className="MuiFormHelperText-root css-1d1r5q-MuiFormHelperText-root"
  >
    error
  </p>,
]
`;
